### PR TITLE
Fixes for editing articles API

### PR DESCRIPTION
**Pull request by UladBohdan**
* `imageUrl` is now required for Articles.
* error codes introduced (in sync with our spreadsheet)
* proper error handling for Bundled API
* fixing a bug with missing data during auth register procedure

### CODE REVIEW

File: src/utils/validation.js [hunk 1]
```diff
@@ -11,12 +11,12 @@ const createArticleValidator = ({ body }, res, next) => {
...
   if (body.locales) {
     Object.entries(body.locales).forEach(([locale, localeData]) => {
-      ['title', 'subtitle', 'slug', 'text', 'locale'].forEach(field => {
+      ['title', 'subtitle', 'slug', 'text'].forEach(field => {
         if (!localeData[field]) {
           set(errors, ['locales', locale, field], 'must be presented');
         }
       });
-      if (locale !== localeData.locale) {
+      if (localeData.locale && locale !== localeData.locale) {
```
Comment (Drapegnik) [inline src/utils/validation.js]: redundant, cause you newer you `localeData.locale`

File: src/api/article/controller.tests.js [hunk 2]
```diff
@@ -342,12 +346,13 @@ describe('Articles Bundled API', () => {
       })
       .expect(400));
 
-  it('should fail to create an article with unsynced locale', () =>
+  it('should fail to create an article with inconsistent locale', () =>
```
Comment (Drapegnik) [inline src/api/article/controller.tests.js]: useless test

File: src/utils/validation.js (hunk 1 above)
Comment (UladBohdan) [reply src/utils/validation.js]: I'd like to keep it in case frontend starts including locales

**Comment by UladBohdan** [comment]
[displayName/auth bug - signing up with lastName does not lead to showing last name in navbar (firstName only). Investigation needed.](https://trello.com/c/E3vjiYaH/183-displayname-auth-bug-signing-up-with-lastname-does-not-lead-to-showing-last-name-in-navbar-firstname-only-investigation-needed)

**Pull request merged**